docs/kubeadm.md

```diff
@@ -0,0 +1,243 @@
+    <master/addons> created essential addon: kube-dns
+
+    Kubernetes master initialised successfully!
+
+    You can connect any number of nodes by running:
+
+    kubeadm join --token <token> <master-ip>
+
+Make a record of the `kubeadm join` command that `kubeadm init` outputs.
+You will need this in a moment.
+The key included here is secret, keep it safe &mdash; anyone with this key will be able to add authenticated nodes to your cluster.
+
+The key is used for mutual authentication between the master and the joining nodes.
+
+### (3/4) Joining your nodes
+
+The nodes are where your workloads (containers and pods, etc) will run.
+If you want to add any new machines as nodes to your cluster, for each machine: SSH to that machine, become root (e.g. `sudo su -`) and run the command that was output by `kubeadm init`.
+For example:
+
+    # kubeadm join --token <token> <master-ip>
+    <util/tokens> validating provided token
+    <node/discovery> created cluster info discovery client, requesting info from "http://138.68.156.129:9898/cluster-info/v1/?token-id=0f8588"
+    <node/discovery> cluster info object received, verifying signature using given token
+    <node/discovery> cluster info signature and contents are valid, will use API endpoints [https://138.68.156.129:443]
+    <node/csr> created API client to obtain unique certificate for this node, generating keys and certificate signing request
+    <node/csr> received signed certificate from the API server, generating kubelet configuration
+    <util/kubeconfig> created "/etc/kubernetes/kubelet.conf"
+
+    Node join complete:
+    * Certificate signing request sent to master and response
+      received.
+    * Kubelet informed of new secure connection details.
+
+    Run 'kubectl get nodes' on the master to see this machine join.
+
+A few seconds later, you should notice that running `kubectl get nodes` on the master shows a cluster with as many machines as you created.
+
+**YOUR CLUSTER IS NOT READY YET!**
+
+Before you can deploy applications to it, you need to install a pod network.
+
+### (4/4) Installing a pod network
+
+You must install a pod network add-on so that your pods can communicate with each other when they are on different hosts.
+**It is necessary to do this before you try to deploy any applications to your cluster.**
+
+Several projects provide Kubernetes pod networks.
+You can see a complete list of available network add-ons on the [add-ons page](/docs/admin/addons/).
+
+You can install [Weave Net](https://github.com/weaveworks/weave-kube) by logging in to the master and running:
+
+    # kubectl apply -f https://git.io/weave-kube
+    daemonset "weave-net" created
+
+If you prefer [Calico](https://github.com/projectcalico/calico-containers/tree/master/docs/cni/kubernetes/manifests/kubeadm) or [Canal](https://github.com/tigera/canal/tree/master/k8s-install/kubeadm), please refer to their respective installation guides.
```

Comment:
@caseydavenport I have added an explicit shout out to Calico and Canal here and linked to their install instructions. If you want me to add a one-liner install command here as well, please let me know which one to use. I don't want to have more than two options in this guide because it starts to become too much cognitive overhead for the user. This is meant to be a quickstart after all :)